examples/SenseDesktop/Program.cs

```diff
@@ -136,37 +136,41 @@ static void Main(string[] args)
 
             //Thread.Sleep(3000);
 
-            //find the bookmark with type
-            var bookmarkExample = new BookmarkExample(app);
-            var task1 = bookmarkExample.ListBookmarksAsync();
-            task1.Wait();
+            //Caluculation Test
+            var calc = new CalculationExample(app);
+            calc.CalcRandom(10);
+
+            ////find the bookmark with type
```

Comment:
@mossbeard 
don't disable / enable sample just with each PR
please make here a consitent naming and all examples should run everytime
all different Examples can be run multithreaded so just add a waitAll in the ende